### PR TITLE
Fixed installation instructions

### PR DESCRIPTION
**Pull request by omichel**
Fixed and simplified installation instructions.

### CODE REVIEW

**Comment by omichel** [comment]
👍

**Comment by DavidMansolino** [comment]
:eyes:

**Comment by omichel** [comment]
Yes, it's documented here: https://github.com/cyberbotics/webots/wiki/Release-Procedure#release-tasks

**Comment by omichel** [comment]
👍

File: docs/guide/installation-procedure.md [hunk 1]
```diff
@@ -27,25 +27,25 @@ Please find instructions in [this section](verifying-your-graphics-driver-instal
 The advantage of this installation is that Webots will be updated automatically with system updates.
 The installation requires the `root` privileges.
 
-First of all, Webots should be authenticated with the [Cyberbotics.asc](https://www.cyberbotics.com/Cyberbotics.asc) signature file which can be downloaded from the [Webots download page](https://www.cyberbotics.com/download), and installed using this command:
+First of all, Webots should be authenticated with the [Cyberbotics.asc](https://cyberbotics.com/Cyberbotics.asc) signature file which can be installed using this command:
 
 ```sh
-curl -s -L https://www.cyberbotics.com/Cyberbotics.asc | sudo apt-key add -
+curl -s -L https://cyberbotics.com/Cyberbotics.asc | sudo apt-key add -
```
[the remark this answers is not in the document]
Comment (omichel) [reply docs/guide/installation-procedure.md]: OK, let's revert to `wget` then...

**Pull request merged**